src/Symfony/Component/DomCrawler/Tests/Field/ChoiceFormFieldTest.php

```diff
@@ -110,6 +110,18 @@ public function testSelects()
         } catch (\InvalidArgumentException $e) {
             $this->assertTrue(true, '->setValue() throws an \InvalidArgumentException if the value is an array');
         }
+
+        $option = $this->createNode('option', 'Hello World', ['value' => 'hello_world']);
```

Comment:
Please create a separate test instead of adding more things in the same test. This provides a much better experience:

- we have a test name telling us what this is about
- the status of the test is separate, giving us better failures

Existing dom-crawler tests are not following the testing best practices because this is one of the oldest components of Symfony, at a time where the usage of phpunit in Symfony was still quite new.